lesson-2/README.md

```diff
@@ -0,0 +1,23 @@
+# Unit 7 Lesson 2: Middleware and Express
+
+## Key Terms
++ Express
++ Middleware
++ Routing
++ JSON
++ Framework
+
+## Essential Questions
+
++ Why might we want to use the Express framework to create our web applications?
++ What is middleware? Why is it useful?
+
+## Learning Assignments
++ [Express Documentation - Getting Started](https://expressjs.com/en/starter/installing.html)
++ [Video Tutorial: Building RESTful APIs with Express](https://www.youtube.com/watch?v=pKd0Rpw7O48) - We'll look at REST in a future lesson, so for now focus mostly on how to use Express
++ [Free Code Camp Guide](https://guide.freecodecamp.org/nodejs/express/)
++ [Middleware Definition](https://developer.mozilla.org/en-US/docs/Glossary/Middleware)
```

Comment:
This is good but throws a lot of content at students at once. It might be a bit overwhelming but will be a good reference nonetheless.